goimports/runner.bash.template

```diff
@@ -11,8 +11,10 @@ mkdir -p "src/$PREFIX_DIR_PATH"
 ln -snf "$BUILD_WORKSPACE_DIRECTORY" "src/$PREFIX_DIR_PATH/$PREFIX_BASE_NAME"
 
 GOPATH=$(pwd -P)
+GOCACHE="$(mktemp -d)"
+trap "rm -rf ${GOCACHE}" EXIT
```

Comment:
Will this work if `GOCACHE` value has spaces? Should it be quoted here? I suspect it should be.

Reply:
Could you try to see if this works `trap "rm -rf '${GOCACHE}'" EXIT`?

Reply:
Done. Works as expected. :+1: